src/code/z_actor.c

```diff
@@ -1889,24 +1912,24 @@ s32 func_800B8500(Actor* actor, PlayState* play, f32 xzRange, f32 yRange, Player
 
     player->talkActor = actor;
     player->talkActorDistance = actor->xzDistToPlayer;
-    player->exchangeItemId = exchangeItemId;
+    player->exchangeItemAction = exchangeItemAction;
 
     CutsceneManager_Queue(CS_ID_GLOBAL_TALK);
     return true;
 }
 
-s32 func_800B85E0(Actor* actor, PlayState* play, f32 radius, PlayerItemAction exchangeItemId) {
-    return func_800B8500(actor, play, radius, radius, exchangeItemId);
+s32 Actor_OfferTalkExchangeRadius(Actor* actor, PlayState* play, f32 radius, PlayerItemAction exchangeItemAction) {
+    return Actor_OfferTalkExchange(actor, play, radius, radius, exchangeItemAction);
 }
 
-s32 func_800B8614(Actor* actor, PlayState* play, f32 radius) {
-    return func_800B85E0(actor, play, radius, PLAYER_IA_NONE);
+s32 Actor_OfferTalk(Actor* actor, PlayState* play, f32 radius) {
+    return Actor_OfferTalkExchangeRadius(actor, play, radius, PLAYER_IA_NONE);
 }
 
-s32 func_800B863C(Actor* actor, PlayState* play) {
+s32 Actor_OfferTalkNearby(Actor* actor, PlayState* play) {
     f32 cylRadius = actor->colChkInfo.cylRadius + 50.0f;
 
-    return func_800B8614(actor, play, cylRadius);
+    return Actor_OfferTalk(actor, play, cylRadius);
```

Comment:
The comment added is good, but I do still kinda think that "nearby" doesnt describe it all to well. Its only nearby if the cylinder radius is small. Are you assuming the colcheckinfo cylinder radius is kept small/as default? cause it is configurable

Reply:
At least for me, even if the collider is huge I would still consider the Player nearby the actor if they are just outside the collider reach (I kind of consider the collider as part of the actor, even if the part drawn is much smaller than the collider). I also like `Nearby` as it mirrors the GetItem function in purpose, though granted the implementation details are different.

Reply:
This isnt the actors collider, to be clear. its an unrelated cylinder that only gets used for a few niche things. mainly talking (as seen here) and some actors use it to avoid projectiles. It can be configured to be much larger than the actual collider cause its seperate

Reply:
Ah, that was my disconnect. Hmm, I can possibly think of a better name. Do you have any suggestions though?

Reply:
I dont, other than just explicitly saying ColChkInfoCyl

Reply:
If we come up with a better name for this cylinder it can be named after it. But the issue is the cylinder doesnt have an exact purpose. Its just a misc cylinder radius and height that lives in the actor struct and the actor system will sometimes use it if it feels like. Its pretty half baked overall

Reply:
Does the full name of the collider `ColChkInfoCyl` need to be in the name or could `Actor_OfferTalkNearColChkInfoCylinder` be simplified to `Actor_OfferTalkNearCol` and just leave it to the summary above the function to specify which of the actor colliders it uses?

Reply:
Hmm, not sure we would want `NearCol` cause the first thing I think of is `NearCollider` which from what I understand above isn't really that. Perhaps `Actor_OfferTalkNearCylinder` with the comment explaining the cylinder it is referring to?

Reply:
I feel like one could easily assume they know it means the collider cylinder and then end up being wrong. Being fully explicit here seems like the best option imo, and maybe one day we'll have a better overall name for this cylinder

Reply:
this conversation needs to happen outside of this PR, but I could see something like "interact cylinder" being used to describe it (think of its main uses, talking and projectile detection)
but have not put any thought into it and double checked all uses. just giving an example of the direction it could go

Reply:
That does seem appropriate, but yeah outside the scope of this PR ( side note `Actor_OfferTalkNearInteractCylinder` does seem nice :) ).

Reply:
Alright, that's fair. We can revisit this along side this collider docs